requirements.txt

```diff
@@ -1,25 +1,20 @@
-# A lot of Great Expectations' codebase and tests are tied to specific databases, cloud providers, etc.
-# In this file, comments on each row indicate which tests that package is used for.
-# To run tests for smaller subsets of infrastructure, please look at other requirements-dev-*.txt files.
-# Otherwise (i.e., if/when you are not concerned with running tests), please ignore these comments.
-
-altair>=4.0.0,<5  # package
-Click>=7.1.2  # package
-importlib-metadata>=1.7.0 # package (included in Python 3.8 by default.)
-ipywidgets>=7.5.1  # package
-jinja2>=2.10  # package
-jsonpatch>=1.22  # package
-jsonschema>=2.5.1  # package
-mistune>=0.8.4,<2.0.0  # package
-numpy>=1.14.1  # package
-pandas>=0.23.0  # package
-pyparsing>=2.4,<3  # package
-python-dateutil>=2.8.1  # package
-pytz>=2015.6  # package
-requests>=2.20  # package
-ruamel.yaml>=0.16,<0.17.18  # package
-scipy>=0.19.0  # package
-termcolor>=1.1.0  # package
-tqdm>=4.59.0  # package
+altair>=4.0.0,<5
+Click>=7.1.2
+importlib-metadata>=1.7.0 # (included in Python 3.8 by default.)
+ipywidgets>=7.5.1
+jinja2>=2.10
+jsonpatch>=1.22
+jsonschema>=2.5.1
+mistune>=0.8.4,<2.0.0
+numpy>=1.14.1
+pandas>=0.23.0
+pyparsing>=2.4,<3
+python-dateutil>=2.8.1
+pytz>=2015.6  # <---- should we update to something in the recent few years?
```

Comment:
Can we resolve this before merging?

Reply:
Comment was added so it could be discussed in PR